src/org/exist/xquery/functions/util/Eval.java

```diff
@@ -147,7 +136,8 @@
             new QName("eval-async", UtilModule.NAMESPACE_URI, UtilModule.PREFIX),
             "Dynamically evaluates an XPath/XQuery expression asynchronously. The ID of the executing thread is returned.",
             new SequenceType[] { EVAL_ARGUMENT },
-            RETURN_NODE_TYPE),
+            RETURN_NODE_TYPE,
+            "The implementation of eval-async was unreliable. eval-async now defers to eval!"),
```

Comment:
i did not realise that the implementation is not stable.....

Reply:
It would stabily screw you up ;-)